add_nyt_bestseller_tag.py

```diff
@@ -20,87 +20,107 @@
 """
 
 import json
-import requests
 
+import requests
 from olclient.bots import AbstractBotJob
 
 
 class AddNytBestsellerJob(AbstractBotJob):
-    @staticmethod
-    def need_to_add_nyt_bestseller_tag(work)-> bool:
+    def __init__(self):
+        super().__init__(job_name='AddNytBestseller')
+
+    def need_to_add_nyt_bestseller_tag(self, work) -> bool:
         """Returns False if the book already has
         a tag that starts with 'nyt:'"""
-        for subj in work['subjects']:
-            if subj.startswith(('nyt:', 'nytimes:')):
-                return False
-        return True
+        try:
+            for subj in work.subjects:
+                if subj.startswith(('nyt:')):
+                    return False
+            return True
```

Comment:
This works but just for your knowledge the more _pythonic_ way of doing this would be something like
`return any(subj.startswith(('nyt:')) for subj in work.subjects)`
^^ double check that works before using it

Reply:
Did not work as expected unfortunately. I'll try to find out what is wrong there.